Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install curl unzip build-essential openjdk-8-jdk python3-dev python3-pip qemu-user -y --no-install-recommends && rm -rf /var/lib/apt/lists/*
+RUN curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.4.0/bazelisk-linux-amd64 > /usr/local/bin/bazelisk && chmod +x /usr/local/bin/bazelisk
```

Comment:
I'd prefer to call it `bazelisk` like we do on CI to keep things consistent

Reply:
Fair enough. I think `bazel` is nicer literally just to save typing while debugging (plus I can't think of any reason anyone should ever use raw bazel instead of bazelisk, the features of the latter should exist in the former), but we can update that in the future both in CI and here to keep things consistent.